frontend/src/MyApp.js

```diff
@@ -12,9 +19,17 @@ function MyApp() {
 
       <Routes>
         <Route path="/" element={<Home />} />
+        <Route path="/home" element={<Home/>} />
         <Route path="pages/:id" element={<RecipePage />} />
         <Route path="/Input" element={<Form />} />
         <Route path="/search/:id" element={<RecipePage />} />
+        <Route path="/login" element={<Login/>} />
```

Comment:
Is there any way to force the user to sign up / log in before reaching the rest of the page?

Reply:
Yes, we can do it by modifying the navbar.js. Right now, the user can't see the profile page and the user page before they signin

Reply:
Cool, yeah what you have right now is great! I was just curious